android-whitelist.json

```
@@ -1376,6 +1376,11 @@
       "group": "com\\.mercadolibre\\.android\\.mldashboard",
       "name": "mldashboard",
       "version": "1\\.\\+"
+    },
+    {
+      "group": "com\\.smartlook\\.recording", 
+      "name": "smartlook",
```

```suggestion
      "name": "app",
```

Siendo que luego vas a hacer `implementation "com.smartlook.recording:app:$smartlookVersion"` entonces debería decir `app` y no `smartlook`.